docker-compose.yml

```
@@ -23,6 +23,6 @@ services:
       - GOOGLE_APPLICATION_CREDENTIALS=${GOOGLE_APPLICATION_CREDENTIALS}
       - GS_BUCKET_NAME=${GS_BUCKET_NAME}
     command: |
-      bash -c "uv run python manage.py makemigrations &&
-      uv run python manage.py migrate &&
-      uv run python manage.py runserver 0.0.0.0:8000"
\ No newline at end of file
+      bash -c "python manage.py makemigrations &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)